src/components/Settings/Screen/Section.js

```diff
@@ -133,6 +141,12 @@ class Section extends React.Component {
 			</React.Fragment>
 		);
 	}
+	
+	handleChange(e) {
+		e.target.key_path = e.target.id;
+		e.target.selection = true;
+		this.props.changeCallback(e.target, e.target);
```

Comment:
Please align equal signs and leave a blank space between the assignment code and the method invocation code.